catlas/filters.py

```diff
@@ -86,7 +88,7 @@ def bulk_filter(config, dask_df, sankey, initial_bulks):
                         meta=("element_in_group", "bool"),
                     )
                 ]
-            elif name == "filter_by_pourbaix_stability":
+            elif name == "filter_by_pourbaix_stability" and "bulk_id" in columns:
```

Comment:
Same

Reply:
same

Reply:
original question was 'why is this necessary'?

Reply:
Because I am using bulk_filter now instead of having separate filters for filtering the df that gets used for parity? I am confused